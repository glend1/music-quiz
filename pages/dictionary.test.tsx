import { render } from "@testing-library/react"
import Dictionary from "./dictionary"

describe("dictionary", () => {
    it("Should render the component", () => {
        const dictionary = render(<Dictionary/>)
        expect(dictionary.baseElement.outerHTML).toMatchSnapshot()
    })
})