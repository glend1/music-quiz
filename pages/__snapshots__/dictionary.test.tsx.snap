// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dictionary Should render the component 1`] = `"<body><div><button class="float">Add</button><section class="container"><div class="card"><h2>Chord</h2><span class="bold">Select a Chord</span><div><svg class="clickable" width="400" viewBox="0 0 400 150"><g width="28.571428571428573"><path class="whiteKey key " x="0" data-natural="C" data-octave="lower" rx="2.8571428571428577" ry="7.5" d="M 0 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="14.285714285714286" y="140" text-anchor="middle">C</text></g><g width="28.571428571428573"><path class="whiteKey key " x="28.571428571428573" data-natural="D" data-octave="lower" rx="2.8571428571428577" ry="7.5" d="M 28.571428571428573 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="42.85714285714286" y="140" text-anchor="middle">D</text></g><g width="14.285714285714286"><path class="blackKey key " x="21.42857142857143" data-sharp="C#" data-flat="Db" data-octave="lower" rx="2.8571428571428577" ry="7.5" d="M 21.42857142857143 0 v 86.25 q 0 7.5 2.8571428571428577 7.5 h 8.571428571428571 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -86.25 Z"></path></g><g width="28.571428571428573"><path class="whiteKey key " x="57.142857142857146" data-natural="E" data-octave="lower" rx="2.8571428571428577" ry="7.5" d="M 57.142857142857146 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="71.42857142857143" y="140" text-anchor="middle">E</text></g><g width="14.285714285714286"><path class="blackKey key " x="50" data-sharp="D#" data-flat="Eb" data-octave="lower" rx="2.8571428571428577" ry="7.5" d="M 50 0 v 86.25 q 0 7.5 2.8571428571428577 7.5 h 8.571428571428571 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -86.25 Z"></path></g><g width="28.571428571428573"><path class="whiteKey key " x="85.71428571428572" data-natural="F" data-octave="lower" rx="2.8571428571428577" ry="7.5" d="M 85.71428571428572 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="100.00000000000001" y="140" text-anchor="middle">F</text></g><g width="28.571428571428573"><path class="whiteKey key " x="114.28571428571429" data-natural="G" data-octave="lower" rx="2.8571428571428577" ry="7.5" d="M 114.28571428571429 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="128.57142857142858" y="140" text-anchor="middle">G</text></g><g width="14.285714285714286"><path class="blackKey key " x="107.14285714285714" data-sharp="F#" data-flat="Gb" data-octave="lower" rx="2.8571428571428577" ry="7.5" d="M 107.14285714285714 0 v 86.25 q 0 7.5 2.8571428571428577 7.5 h 8.571428571428571 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -86.25 Z"></path></g><g width="28.571428571428573"><path class="whiteKey key " x="142.85714285714286" data-natural="A" data-octave="lower" rx="2.8571428571428577" ry="7.5" d="M 142.85714285714286 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="157.14285714285714" y="140" text-anchor="middle">A</text></g><g width="14.285714285714286"><path class="blackKey key " x="135.71428571428572" data-sharp="G#" data-flat="Ab" data-octave="lower" rx="2.8571428571428577" ry="7.5" d="M 135.71428571428572 0 v 86.25 q 0 7.5 2.8571428571428577 7.5 h 8.571428571428571 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -86.25 Z"></path></g><g width="28.571428571428573"><path class="whiteKey key " x="171.42857142857144" data-natural="B" data-octave="lower" rx="2.8571428571428577" ry="7.5" d="M 171.42857142857144 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="185.71428571428572" y="140" text-anchor="middle">B</text></g><g width="14.285714285714286"><path class="blackKey key " x="164.2857142857143" data-sharp="A#" data-flat="Bb" data-octave="lower" rx="2.8571428571428577" ry="7.5" d="M 164.2857142857143 0 v 86.25 q 0 7.5 2.8571428571428577 7.5 h 8.571428571428571 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -86.25 Z"></path></g><g width="28.571428571428573"><path class="whiteKey key " x="200.00000000000003" data-natural="C" data-octave="higher" rx="2.8571428571428577" ry="7.5" d="M 200.00000000000003 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="214.2857142857143" y="140" text-anchor="middle">C</text></g><g width="28.571428571428573"><path class="whiteKey key " x="228.5714285714286" data-natural="D" data-octave="higher" rx="2.8571428571428577" ry="7.5" d="M 228.5714285714286 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="242.8571428571429" y="140" text-anchor="middle">D</text></g><g width="14.285714285714286"><path class="blackKey key " x="221.42857142857147" data-sharp="C#" data-flat="Db" data-octave="higher" rx="2.8571428571428577" ry="7.5" d="M 221.42857142857147 0 v 86.25 q 0 7.5 2.8571428571428577 7.5 h 8.571428571428571 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -86.25 Z"></path></g><g width="28.571428571428573"><path class="whiteKey key " x="257.14285714285717" data-natural="E" data-octave="higher" rx="2.8571428571428577" ry="7.5" d="M 257.14285714285717 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="271.42857142857144" y="140" text-anchor="middle">E</text></g><g width="14.285714285714286"><path class="blackKey key " x="250.00000000000006" data-sharp="D#" data-flat="Eb" data-octave="higher" rx="2.8571428571428577" ry="7.5" d="M 250.00000000000006 0 v 86.25 q 0 7.5 2.8571428571428577 7.5 h 8.571428571428571 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -86.25 Z"></path></g><g width="28.571428571428573"><path class="whiteKey key " x="285.7142857142857" data-natural="F" data-octave="higher" rx="2.8571428571428577" ry="7.5" d="M 285.7142857142857 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="300" y="140" text-anchor="middle">F</text></g><g width="28.571428571428573"><path class="whiteKey key " x="314.2857142857143" data-natural="G" data-octave="higher" rx="2.8571428571428577" ry="7.5" d="M 314.2857142857143 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="328.57142857142856" y="140" text-anchor="middle">G</text></g><g width="14.285714285714286"><path class="blackKey key " x="307.14285714285717" data-sharp="F#" data-flat="Gb" data-octave="higher" rx="2.8571428571428577" ry="7.5" d="M 307.14285714285717 0 v 86.25 q 0 7.5 2.8571428571428577 7.5 h 8.571428571428571 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -86.25 Z"></path></g><g width="28.571428571428573"><path class="whiteKey key " x="342.85714285714283" data-natural="A" data-octave="higher" rx="2.8571428571428577" ry="7.5" d="M 342.85714285714283 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="357.1428571428571" y="140" text-anchor="middle">A</text></g><g width="14.285714285714286"><path class="blackKey key " x="335.7142857142857" data-sharp="G#" data-flat="Ab" data-octave="higher" rx="2.8571428571428577" ry="7.5" d="M 335.7142857142857 0 v 86.25 q 0 7.5 2.8571428571428577 7.5 h 8.571428571428571 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -86.25 Z"></path></g><g width="28.571428571428573"><path class="whiteKey key " x="371.4285714285714" data-natural="B" data-octave="higher" rx="2.8571428571428577" ry="7.5" d="M 371.4285714285714 0 v 142.5 q 0 7.5 2.8571428571428577 7.5 h 22.857142857142858 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -142.5 Z"></path><text class="whiteKeyText" x="385.71428571428567" y="140" text-anchor="middle">B</text></g><g width="14.285714285714286"><path class="blackKey key " x="364.2857142857143" data-sharp="A#" data-flat="Bb" data-octave="higher" rx="2.8571428571428577" ry="7.5" d="M 364.2857142857143 0 v 86.25 q 0 7.5 2.8571428571428577 7.5 h 8.571428571428571 q 2.8571428571428577 0 2.8571428571428577 -7.5 v -86.25 Z"></path></g></svg></div><div><div>Click a Key to begin Chord selection</div></div><button class="deleteParent">Remove</button></div></section></div></body>"`;
